oshi-core/src/main/java/oshi/software/os/OperatingSystem.java

```diff
@@ -372,8 +373,8 @@ default List<OSDesktopWindow> getDesktopWindows(boolean visibleOnly) {
      *
      * @return A list of installed applications or an empty list if unsupported.
      */
-    default List<ApplicationInfo> getInstalledApplications() {
-        return Collections.emptyList();
+    default Set<ApplicationInfo> getInstalledApplications() {
+        return Collections.EMPTY_SET;
```

Comment:
This will go back to `.emptyList()` but for future reference, the constants don't infer types so it's preferable to use `Collections.emptySet()`.  I think what you have, without a cast, generates a compiler warning about an unchecked conversion.